io.github.archisman_panigrahi.typhoon.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id" : "io.github.archisman_panigrahi.typhoon",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "48",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-version" : "48",
+    "command" : "typhoon",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=all",
```

Comment:
why does a weather app needs "device=all"

Reply:
I wrongly assumed it was necessary to indicate both desktop and mobile phones. Now I've realized that this is not necessary.